doc/sphinxdoc/models.rst

```diff
@@ -0,0 +1,653 @@
+.. Essentia TensorFlow models
+
+Essentia TensorFlow models
+==========================
+
+This is a list of pre-trained TensorFlow models available in Essentia for various (music) audio analysis and classification tasks. To use the models in Essentia, see the `machine learning inference <machine_learning.html>`_ page.
+
+
+All the models created by the MTG are licensed under CC BY-NC-SA 4.0 (https://creativecommons.org/licenses/by-nc-sa/4.0/) and are also available under proprietary license upon request (https://www.upf.edu/web/mtg/contact).
```

Comment:
created -> trained?

Reply:
This does not give additional details on the origin of the models.